packages/localizations/src/en-US.ts

```diff
@@ -456,6 +512,7 @@ export const enUS: LocalizationResource = {
   socialButtonsBlockButton: 'Continue with {{provider|titleize}}',
   socialButtonsBlockButtonManyInView: '{{provider|titleize}}',
   unstable__errors: {
+    already_a_member_in_organization: '{{email}} já é membro da organização.',
```

Comment:
@LauraBeatris this should be in english, i believe this was added to create the images in the description of the PR